### PR TITLE
Add GitHub link

### PR DESCRIPTION
**Pull request by maelle**
for easier parsing by codemetar for the registry

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -33,7 +33,7 @@ Imports:
     stats,
     tibble,
     rlang (>= 0.2.0)
-URL: http://visdat.njtierney.com/
+URL: http://visdat.njtierney.com/, https://github.com/ropensci/visdat
```
Comment (njtierney) [inline DESCRIPTION]: Looks good to me! :)

**Comment by njtierney** [comment]
Thanks @maelle :)

**Comment by njtierney** [comment]
What is the registry, btw?

**Comment by maelle** [comment]
The registry is used in e.g https://ropensci.org/packages/

It is curated by hand by @sckott in https://github.com/ropensci/roregistry

But will soon be generated via the use of https://github.com/ropensci/codemetar on all repos (I saw the missing link here when working on this)

**Pull request merged**